### ISSUE TITLE
Why silence the `console-message` event for offscreen webcontents?

### ISSUE THREAD

**Issue by brian-mann** [electron/electron#11917]
Specifically this line of code here: https://github.com/electron/electron/blob/9bd7d53cf85ac36e81a080c4fd0572ca34d395ff/atom/browser/api/atom_api_web_contents.cc#L484

Just wondering what the ramifications or reasoning is behind this?

At https://github.com/cypress-io/cypress we'd like to capture these for debugging purposes, and we primarily run in `offscreen` mode.

We could use the internal chromium logger, but it doesn't stringify objects so it's fairly useless. All you'll see is `[object Object]`.

**Comment by codebytere** [comment]
Thanks for reaching out!

Because we treat our issues list as the team's backlog, we close issues that are questions since they don't represent a task needing to be completed. For most questions about Electron there are a lot of options. 
Check out the [Electron community](https://github.com/electron/electron#community). There are also a bunch of helpful people in this [community forum](https://discuss.atom.io/c/electron) that should be willing to point you in the right direction.

**Comment by brian-mann** [comment]
@codebytere I understand - but in this case - my question is referring to a technical implementation that's been in discussion before in several PR's.

I looked through the PR's associated to this change - and even the tests, and thus far I cannot find any commentary on why it's being done like this - and more importantly why it cannot be done with offscreen rendering.

Would it help if I opened a different issue or rephrased the issue from being a question to instead asking for this change to be implemented as a feature or enhancement? Then it would be a task needing to be completed.
